hello.c

```diff
@@ -2,7 +2,8 @@
 
 int main()
 {
-    printf("Hello, World\n");
+    printf("Oh yeah, Tsoding and Strager are just the absolute best streamers of all time...\n");
```

Comment:
Who is Strager?

Reply:
Who is Strager? Only the most entertaining tech streamer on the internet! You're missing out if you haven't checked out his streams yet.

Reply:
really?  I haven't heard of him.  I'm more of a _s_trager fan myself